modules/ROOT/pages/apikit-asyncapi-avro-support.adoc

```diff
@@ -0,0 +1,63 @@
+= Implementing and Validating Avro Schemas in APIkit for AsyncAPI Module
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+APIkit for AsyncAPI supports Avro, a data serialization system that helps you define the format for specifying the structure and encoding of messages. This module shows you the structure of messages in the design phase. APIkit for AsyncAPI also supports xref:studio::datasense-concept.adoc[DataSense] of messages with Avro schema.
+
+== Schema Definitions
+
+The Avro spec defines its format by giving the value of `schemaFormat` as `application/vnd.apache.avro`.
+
+The Avro spec can describe the schema inline:
+
+----
+publish:
+  summary: inform details about person
+  operationId: lightMeasured
+  message:
+    schemaFormat: 'application/vnd.apache.avro;version=1.9.0'
+    payload:
+      name: lightMeasured
+      type: record
+      fields:
+        - name: lumens
+          type: int
+          example: 134
+        - name: sentAt
+          type: string
+          logicalType: timestamp-millis
+          default: 10
```

Comment:
sorry, remove this `default`

Reply:
Please, remember that you can add a suggestion and edit the doc yourself. 
Once done, I can commit to your suggestion.
Thanks!